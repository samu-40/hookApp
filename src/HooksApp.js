import React from 'react'
// import PropTypes from 'prop-types'

const HooksApp = props => {
    return (
        <div>
            <h1>Hola mundo</h1>
        </div>
    )
}

// HooksApp.propTypes = {}

export default HooksApp
