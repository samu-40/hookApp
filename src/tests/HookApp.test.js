import { shallow } from "enzyme";
import HooksApp from "../HooksApp";

describe('Pruebas en HookApp.js', () => {
    test('Debe mostrar correctamente el componente', () => {
        const wrapper = shallow(<HooksApp/>);

        expect(wrapper).toMatchSnapshot();
    })
    
})
