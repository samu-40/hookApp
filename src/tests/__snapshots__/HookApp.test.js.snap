// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HookApp.js Debe mostrar correctamente el componente 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
